governance/reviews/handbook.md

```diff
@@ -0,0 +1,49 @@
+# Governance Reviewer Role Handbook
+
+WIP: This guide is not complete, but the instructions it contains are a starting point.
+
+# Goals
+
+The goal of the Governance Review process is to help projects improve their own governance.  As a reviewer, it is your goal to:
+
+1. Perform one or more Governance Reviews in order to help projects complete their requirements for matriculation  
+2. Assist projects in identifying areas where their governance could use improvement and carrying out that improvement  
+
+A successful Governance Review results in projects discovering, and fixing, issues around their governance processes or documentation, and to make a plan to gradually improve their governance continuously over the life of the project.
```

Comment:
```suggestion
A successful Governance Review results in projects discovering, and fixing, issues around their governance processes or documentation, and to make a plan to gradually improve their governance continuously over the life of the project for their benefit and healthy longevity.
```

Reply:
I see what you're getting at here, but that feels awkward.  What about:

```
A successful Governance Review results in projects discovering, and fixing, issues around their governance processes or documentation.  It also helps enable them to make a plan to gradually improve their governance for the project's long-term health.
```